### DISCUSSION TITLE
Em thấy gõ font không được ngay ngắn, phải làm sao ạ?

### DISCUSSION THREAD

**Discussion by tran184** [BambooEngine/ibus-bamboo#350] [Q&A] [answered]
Ví dụ như em gõ trên Discord:
![Screenshot from 2022-08-10 22-58-46](https://user-images.githubusercontent.com/30346378/183957329-966ab408-78e9-4852-ad22-9ea2d14b3aa9.png)

Em đang để cài đặt mặc định, chỉ thay đổi kiểu gõ sang Telex 2 thôi, bảng mã vẫn là Unicode.
Có phải là do font trên Discord gây ra lỗi không ạ? Hay là có nguyên nhân khác ạ? Có giải pháp không ạ, nếu có thì giải pháp là gì ạ?

**Comment by MeowIce** [comment, marked as answer]
> Ví dụ như em gõ trên Discord:
> ![Screenshot from 2022-08-10 22-58-46](https://user-images.githubusercontent.com/30346378/183957329-966ab408-78e9-4852-ad22-9ea2d14b3aa9.png)
> 
> Em đang để cài đặt mặc định, chỉ thay đổi kiểu gõ sang Telex 2 thôi, bảng mã vẫn là Unicode.
> Có phải là do font trên Discord gây ra lỗi không ạ? Hay là có nguyên nhân khác ạ? Có giải pháp không ạ, nếu có thì giải pháp là gì ạ? 
> 

Này là do font mặc định của Discord nó thế. Nếu bạn thấy ngứa mắt thì cài thêm BetterDiscord rồi tự đổi sang font khác cho hợp ý bạn nha.
